src/scss/CustomDropdownToggle.scss

```diff
@@ -0,0 +1,27 @@
+.btn-primary:not(:disabled):not(.disabled):active#custom-toggle,
```

Comment:
ok so this allows us to overwrite original bootstrap styling in just one place?

Reply:
This only changes the styling of primary buttons with the id "custom-toggle" to prevent other buttons from being re-styled as a side effect